testdata/p4_16_samples/runtime-index-bmv2.stf

```diff
@@ -0,0 +1,10 @@
+packet abababab00000002000000010000000200000003
+expect abababab000000020000000100000002abababab
```

Comment:
I would recommend putting a `$` at the end of each `expect` line, to cause the STF test to verify that the output packet is no longer than the one shown in the `expect` line, and causing the test to fail if the actual output packet is longer than the expected one.

Reply:
Will do.  I have been away from STF for a long time and didn't add port.  I will fix the Travis error.

Reply:
You are also missing input port numbers on the packet lines, and output port numbers on the expect lines.

Reply:
I already said in my reply, I didn't add port - I added it.

I am using latest behavioral-model code which has Antonin's l-value changes to runtime index.  But bmv2 test with simple_switch fails (this is the Travis error).   I have narrowed down the error to be caused the the line of P4 code using a runtime index. 

```
hemant@ubuntu:/tmp/p4c/build$ bmv2/testdata/p4_16_samples/runtime-index-bmv2.p4.test -f
Check for  /tmp/p4c/testdata/p4_16_samples/runtime-index-bmv2.stf
Calling target program-options parser
terminate called after throwing an instance of 'std::runtime_error'
  what():  in Json::Value::operator[](char const*)const: requires objectValue
*** Test failed
hemant@ubuntu:/tmp/p4c/build$ 
```